resources/servo.css

```diff
@@ -7,6 +7,7 @@ input[type="reset"]     { background: lightgrey; border-top: solid 1px #EEEEEE;
 input[type="hidden"]    { display: none !important }
 input[type="checkbox"],
 input[type="radio"]     { font-family: monospace !important; border: none !important; background: transparent; }
+input[dir="rtl"]        { text-align: right; }
```

Comment:
Oh I have tried this at first. But when I run a test html with `<input value="test" dir="rtl">` or `<input value="test" dir="auto">`, It didn't rendered as aligned to right and i thought it didn't worked. That's why I changed to `input[dir="rtl"]`. Now, I'm looking the rendered html in unsuccessfull test with `./mach run -d tests/wpt/web-platform-tests/html/dom/elements/global-attributes/dir_auto-input-N-R.html` but it is still aligned to left with `input:dir(rtl)`